i18n/locales/en.json

```
@@ -67,5 +67,8 @@
   "components.sections.newHeader.links.apiDocs": "API Docs",
   "components.sections.newHeader.links.about": "About",
   "components.sections.newHeader.links.download": "Downloads",
-  "components.sections.newHeader.links.certification": "Certification"
+  "components.sections.newHeader.links.certification": "Certification",
+  "components.downloadToggle.current": "Current",
```

keys should be as per  [translation guidelines](https://github.com/nodejs/nodejs.org/blob/572c1edd3eaaf32c6f803894ad54b9e4b219110d/TRANSLATION.md?plain=1#L48)
```suggestion
  "components.downloads.downloadToggle.current": "Current",
```